Resources/Prototypes/DeltaV/Catalog/uplink_catalog.yml

```diff
@@ -9,6 +9,17 @@
   categories:
   - UplinkUtility
 
+- type: listing
+  id: Decimator Railgun Bundle
+  name: uplink-decimator-bundle-name
+  description: uplink-decimator-bundle-desc
+  productEntity: ClothingBackpackDuffelSyndicateFilledRailgun
+  icon: { sprite: DeltaV/Objects/Weapons/Guns/Railguns/railgun.rsi, state: icon }
+  cost:
+    Telecrystal: 20
```

Comment:
the Hristov, a slightly worse but far more concealable gun (that also comes with more ammo, and with optional ammo in the uplink) costs only 12 TC. 20 is astronomical.